OJP/OJP_Trips.xsd

```diff
@@ -756,27 +756,37 @@
 			</xs:element>
 			<xs:element name="GuidanceAdvice" type="GuidanceAdviceEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>various types of guidance advice given to travelle.</xs:documentation>
+					<xs:documentation>Various types of guidance advice given to traveller.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="TurnAction" type="TurnActionEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>the range of alternative turns that can be described.</xs:documentation>
+					<xs:documentation>The range of alternative turns that can be described.</xs:documentation>
```

Comment:
I think possible might be more fitting?
"The range of possible turns that can be described."